pkg/sentry/loader/elf.go

```diff
@@ -560,6 +566,13 @@ func loadInitialELF(ctx context.Context, m *mm.MemoryManager, fs *cpuid.FeatureS
 		return loadedELF{}, nil, err
 	}
 
+	// Check Image Compatibility.
+	ok := strings.EqualFold(arch.Host.String(), info.arch.String())
```

Comment:
Why isn't this just arch.Host == info.arch?

Reply:
Ensure sufficient robustness. Do you want to change it?

Reply:
Done.